src/plugins/intel_gpu/src/kernel_selector/kernels/reorder/reorder_weights_int4.cpp

```diff
@@ -74,6 +81,7 @@ bool ReorderWeightsKernelInt4::Validate(const Params& params) const {
     supported_case |= input.GetLayout() == WeightsLayout::oiyx && output.GetLayout() == WeightsLayout::os_iyx_osv64;
     supported_case |= input.GetLayout() == WeightsLayout::oiyx && output.GetLayout() == WeightsLayout::os_is_yx_osv64_isv2;
     supported_case |= input.GetLayout() == WeightsLayout::ioyx && output.GetLayout() == WeightsLayout::oiyx;
+    supported_case |= input.GetLayout() == WeightsLayout::oiyx && output.GetLayout() == WeightsLayout::oiyx && has_only_inner_most_padding;
```

Comment:
This kernel works only when input size is odd and we have stride in even number. But this condition check is different..

Reply:
Change checking condition for this case. thanks.